apps/vr-tests-react-components/src/stories/Rating.stories.tsx

```diff
@@ -48,11 +48,11 @@ storiesOf('Rating Converged', module)
   ))
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=30666&commitId=53114653271539c972720380847b40ef01ab13b0&env=prod&clientType=FLUENTUI)


  <details><summary><b>Rating Converged</b> 24 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Rating Converged.Marigold Rating with half star - High Contrast.rest.chromium.png|244040| Changed|
|Rating Converged.Brand Rating with half star.rest.chromium.png|247805| Changed|
|Rating Converged.Marigold Rating with half star.rest.chromium.png|248120| Changed|
|Rating Converged.Rating size medium.rest.chromium.png|245658| Changed|
|Rating Converged.Neutral Rating with half star - Dark Mode.rest.chromium.png|243362| Changed|
|Rating Converged.Rating size large.rest.chromium.png|246490| Changed|
|Rating Converged.Rating size extra-large.rest.chromium.png|248084| Changed|
|Rating Converged.Neutral Rating with half star - High Contrast.rest.chromium.png|244362| Changed|
|Rating Converged.Marigold Rating with half star - Dark Mode.rest.chromium.png|243134| Changed|
|Rating Converged.Rating with custom square icons.rest.chromium.png|247440| Changed|
|Rating Converged.Brand Rating with half star - Dark Mode.rest.chromium.png|242899| Changed|
|Rating Converged.Neutral Rating with half star.rest.chromium.png|247648| Changed|
|Rating Converged.Rating with custom circle icons.rest.chromium.png|247372| Changed|
|Rating Converged.Brand Rating with half star - High Contrast.rest.chromium.png|244362| Changed|
|Rating Converged.Rating size small.rest.chromium.png|244812| Changed|
|Rating Converged.Brand Rating with half star - Dark Mode.hover.chromium.png|0| Removed|
|Rating Converged.Brand Rating with half star - High Contrast.hover.chromium.png|0| Removed|
|Rating Converged.Brand Rating with half star.hover.chromium.png|0| Removed|
|Rating Converged.Marigold Rating with half star - Dark Mode.hover.chromium.png|0| Removed|
|Rating Converged.Marigold Rating with half star - High Contrast.hover.chromium.png|0| Removed|
|Rating Converged.Marigold Rating with half star.hover.chromium.png|0| Removed|
|Rating Converged.Neutral Rating with half star - Dark Mode.hover.chromium.png|0| Removed|
|Rating Converged.Neutral Rating with half star - High Contrast.hover.chromium.png|0| Removed|
|Rating Converged.Neutral Rating with half star.hover.chromium.png|0| Removed|

</details>


  <details><summary><b>RatingDisplay Converged</b> 19 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|RatingDisplay Converged.Brand with half value - Dark Mode.rest.chromium.png|242961| Changed|
|RatingDisplay Converged.Brand with half value - High Contrast.rest.chromium.png|243698| Changed|
|RatingDisplay Converged.Compact.rest.chromium.png|245728| Changed|
|RatingDisplay Converged.Marigold with half value.rest.chromium.png|245654| Changed|
|RatingDisplay Converged.Marigold with half value - Dark Mode.rest.chromium.png|242988| Changed|
|RatingDisplay Converged.Marigold with half value - High Contrast.rest.chromium.png|243636| Changed|
|RatingDisplay Converged.Neutral with half value.rest.chromium.png|245607| Changed|
|RatingDisplay Converged.Brand with half value.rest.chromium.png|245605| Changed|
|RatingDisplay Converged.no value.rest.chromium.png|245752| Changed|
|RatingDisplay Converged.Neutral with half value - High Contrast.rest.chromium.png|243698| Changed|
|RatingDisplay Converged.rounded down.rest.chromium.png|245569| Changed|
|RatingDisplay Converged.custom circle icons.rest.chromium.png|245538| Changed|
|RatingDisplay Converged.rounded up.rest.chromium.png|245539| Changed|
|RatingDisplay Converged.Neutral with half value - Dark Mode.rest.chromium.png|243040| Changed|
|RatingDisplay Converged.custom square icons.rest.chromium.png|245547| Changed|
|RatingDisplay Converged.size large with value and count.rest.chromium.png|246140| Changed|
|RatingDisplay Converged.size extra-large with value and count.rest.chromium.png|247307| Changed|
|RatingDisplay Converged.size small with value and count.rest.chromium.png|245566| Changed|
|RatingDisplay Converged.size medium with value and count.rest.chromium.png|245458| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>